azurerm_monitor_private_link_scope - api 2021-07-01-preview - ingestion and query mode access added (#25763)
hashicorp/terraform-provider-azurerm: website/docs/r/monitor_private_link_scope.html.markdown

* `resource_group_name` - (Required) The name of the Resource Group where the Azure Monitor Private Link Scope should exist. Changing this forces a new resource to be created.

> * `ingestion_access_mode` - (Optional) The default ingestion access mode for the associated private endpoints in scope. Possible values `Open` and `PrivateOnly`. Omitting this field sets this value to `Open`.
> 
> * `query_access_mode` - (Optional) The default query access mode for hte associated private endpoints in scope. Possible values `Open` and `PrivateOnly`. Omitting this field sets this value to `Open`.

Reviewer: ```suggestion
* `ingestion_access_mode` - (Optional) The default ingestion access mode for the associated private endpoints in scope. Possible values are `Open` and `PrivateOnly`. Defaults to `Open`.

* `query_access_mode` - (Optional) The default query access mode for hte associated private endpoints in scope. Possible values are `Open` and `PrivateOnly`. Defaults to `Open`.
```
